.github/workflows/benchmark-pipeline-test.yml

```diff
@@ -3,9 +3,6 @@ name: Benchmark Pipeline Test
 on:
   push:
 
-concurrency:
-  group: benchmark
-
```

Comment:
I'm afraid this was copy pasta from the benchmark pipeline workflow.

It is needed there so we only benchmark a single configuration at a time on the equinix cluster. The integration test uses a k3s container so this isn't needed.